packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.test.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import { mount } from 'enzyme';
+
+import { ICalloutProps } from '../../Callout';
+import { DirectionalHint } from '../../common/DirectionalHint';
+import { assign } from '../../Utilities';
+import { TooltipHost } from './TooltipHost';
+import { ITooltipProps, TooltipDelay } from './Tooltip.types';
+
+// tslint:disable:jsx-no-lambda
+
+describe('TooltipHost', () => {
+  it('renders correctly', () => {
+    const component = renderer.create(<TooltipHost />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
I tried making these snapshot tests more meaningful for this PR but decided to address it later, unless someone knows how to easily set internal state with a shallow render now. (The full mount snapshots are over 10MB each.)